mk/xenadmin-build.sh

```diff
@@ -133,6 +133,10 @@ cd ${REPO}/XenAdminTests/bin/ && tar -czf XenAdminTests.tgz ./Release
 cd ${REPO}/XenAdmin/TestResources && tar -cf ${OUTPUT_DIR}/XenCenterTestResources.tar *
 cp ${REPO}/XenAdminTests/bin/XenAdminTests.tgz ${OUTPUT_DIR}/XenAdminTests.tgz
 
+#include cfu validator binary in output directory
+cd ${REPO}/CFUValidator && zip CFUValidator.zip ./bin/Release/*.dll ./bin/Release/CFUValidator.exe ./bin/Release/XenCenterMain.exe
```

Comment:
I think it would be much simpler for the consumer of this zip if you did
```
cd ${REPO}/CFUValidator/bin/Release && zip CFUValidator.zip ./{*.dll,CFUValidator.exe,XenCenterMain.exe}
cp ${REPO}/CFUValidator/bin/Release/CFUValidator.zip ${OUTPUT_DIR}/CFUValidator.zip
```
otherwise you'll get the tree `bin/Release/<files>` in the zip, which I doubt is needed.

Reply:
Good point!